includes/class-newspack-newsletters.php

```diff
@@ -284,6 +294,24 @@ public static function rest_api_init() {
 		);
 	}
 
+	/**
+	 * Retrieve Layouts.
+	 */
+	public static function api_get_layouts() {
+		$layouts_query = new WP_Query(
+			array(
+				'post_type'      => Newspack_Newsletters_Layouts::NEWSPACK_NEWSLETTERS_LAYOUT_CPT,
+				'posts_per_page' => -1,
+			)
+		);
+		$layouts       = array_merge(
+			$layouts_query->get_posts(),
+			Newspack_Newsletters_Layouts::get_default_layouts()
+		);
+
```

Comment:
Could we throw `apply_filters( 'newspack_newsletters_templates', [] )` back in so plugins and themes can add their own layouts?

Reply:
Done in c35826a7f6e6264189d2d1801e01c78ec2e699f9